packages/ketcher-react/src/script/editor/Editor.ts

```diff
@@ -94,14 +99,25 @@ function selectStereoFlagsIfNecessary(
   return stereoFlags
 }
 
-interface Selection {
+export interface Selection {
```

Comment:
I propose to create separate file for types/interfaces

Reply:
[create EditorInterfaces.ts](https://github.com/epam/ketcher/pull/2007/commits/3d9e5975da135202c7ad7abb0286a6ce863a7c42)
[3d9e597](https://github.com/epam/ketcher/pull/2007/commits/3d9e5975da135202c7ad7abb0286a6ce863a7c42)